### ISSUE TITLE
Debugging a single test (It) runs all tests

### ISSUE THREAD

**Issue by mikeblakeuk** [machine/machine.specifications#451]
Running tests appears to work fine, just not debugging

Given VS2019 6.11.3
And R# JetBrains ReSharper 2021.2.1 (Build 212.0.20210826.105310 built on 2021-08-26)
And machine.specifications.runner.resharper 2021.2.1 (Aug 12, 2021)
When I debug a single It (right click)
Then it should only debug the class I'm in
Actual: All tests are run in that namespace

**Comment by robertcoltheart** [comment]
Yep, this is a known issue, and we basically need a better filtering mechanism when sending requests over to the MSpec runner. Unfortunately this isn't immediately solvable without some big changes to the API. Marking this as a 2.0 thing.

**Comment by mikeblakeuk** [comment]
Is it something we could PR for 1.x?
Any news on 2.0.0 ? Can we get a beta version?
Thanks
mike

**Comment by robertcoltheart** [comment]
Nothing yet, will update you when I'm closer

**Comment by robertcoltheart** [comment]
You can use the VS test explorer, this problem only affects ReSharper.
